cmd/satellite/entrypoint

```diff
@@ -25,4 +25,8 @@ fi
 
 RUN_PARAMS="${RUN_PARAMS:-} --config-dir ${CONF_PATH}"
 
+if [ "${SATELLITE_API:-}" = "true" ]; then
+  exec ./satellite run api $RUN_PARAMS "$@"
+fi
```

Comment:
Shouldn't this use `else` to ensure that after `satellite run api` exits it doesn't run `satellite run`?